app/src/components/PageTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import { Helmet } from "react-helmet";
+
+type Props = {
+	title?: string;
+	favicon?: string;
+};
+
+/**
+ *
+ * @param title The page title
+ * @param favicon A custom Favicon for the page you're on
+ */
+const PageTitle = ({ title, favicon }: Props) => {
+	return (
+		<Helmet>
+			<title>
+				{title && `${title} - `}AdamRMS
+			</title>
+			{favicon ? (
+				<link rel="shortcut icon" href={favicon} />
+			) : (
+				<link
+					rel="shortcut icon"
+					type="image/png"
+					href="/assets/icon/favicon.png"
```

Comment:
Not sure if this is the correct path for CDN but it works locally

Reply:
Aren't we trying to package most icons etc locally so there is a bit of offline compatibility?

Reply:
Is this now an outdated comment @Robert-Watts ?

Reply:
yep